sei-db/config/config.go

```diff
@@ -2,7 +2,8 @@ package config
 
 const (
 	DefaultSnapshotInterval          = 10000
-	DefaultSnapshotKeepRecent        = 0 // set to 0 to only keep one current snapshot
+	DefaultSnapshotKeepRecent        = 0    // set to 0 to only keep one current snapshot
+	DefaultSnapshotMinTimeInterval   = 3600 // 1 hour in seconds
```

Comment:
Subjective: I usually write these as multiples of numbers to make it obvious what the value is as a gesture to make code more readable. For example, I would write this as `60*60`. 

A far better approach, would have been to type `SnapshotMinTimeInterval` as `time.Duration`. That way, the value of this constant would have been `time.Hour`. In terms of parsing the value from config, we would have then parsed via `time.time.ParseDuration` which accepts human readable notation like `1h`. I understand there might be weeds here with parsing via cobra TOML library etc. and there is indeed a larger task to untangle all of that which is well beyond the scope of this PR :)